devtools/gradle/gradle-model/src/main/java/io/quarkus/gradle/tooling/GradleApplicationModelBuilder.java

```diff
@@ -116,7 +122,15 @@ public Object buildAll(String modelName, ModelParameter parameter, Project proje
         collectDependencies(classpathConfig.getResolvedConfiguration(), workspaceDiscovery,
                 project, modelBuilder, appArtifact.getWorkspaceModule().mutable());
         collectExtensionDependencies(project, deploymentConfig, modelBuilder);
-
+        if (mvnDeps != null) {
```

Comment:
We shouldn't be doing it this way. Can we remove adding these `mvnDeps` to the model?

Reply:
These would need to be added to the runtime configuration to make sure all the dependencies are properly resolved and available for the application. This kind of change would need more testing and review.

Reply:
Removed.